src/containers/Trends.tsx

```diff
@@ -1,16 +1,9 @@
 import { Container } from "react-bootstrap";
 import { useTranslation } from "react-i18next";
-import styled from "styled-components";
 
 import { TrendItem } from "../components/TrendItem";
 import { MockTrend } from "../types/mock-api-types";
-
-const StyledLink = styled.a`
-  transition: 0.3s;
-  :hover {
-    background-color: rgba(0, 0, 0, 0.03);
-  }
-`;
+import { LinkWithLanguageQueryParam } from "./LinkWithLanguageQueryParam";
```

Comment:
If you want to style this link you can write: 

const StyledLink = styled(LinkWithLanguageQueryParam)`
      // stylisation
   }
 `;